plugins/irma-server/index.js

```diff
@@ -16,6 +16,13 @@ module.exports = class IrmaServer {
         return this._startNewSession();
       case 'MediumContemplation':
         return this._startWatchingServerState(payload);
+      case 'Cancelled':
+      case 'TimedOut':
+      case 'Error':
+        // If session cannot be restarted, the error state is permanent. Therefore abort the flow.
+        if ( !this._options.session.start )
+          return this._stateMachine.transition('abort', 'No restart possible');
```

Comment:
Zoals genoemd in #30: is het hergebruik van de `abort` state de beste manier om dit te doen..?

Reply:
Ik zal in de andere pull request nog even kijken naar de naamgeving en dan kijken hoe dat hierin doorwerkt.